app/src/main/ipcListeners.ts

```diff
@@ -144,6 +167,30 @@ export function attachIpcListeners() {
     }
   );
 
+  ipcMain.handle(
+    'query-avg-length',
+    async (event, filters: SharedQueryFilters) => {
+      const db = getDb();
+      return queryAverageMessageLength(db, filters);
+    }
+  );
+
+  ipcMain.handle(
+    'query-double-texts',
+    async (event, filters: SharedQueryFilters) => {
+      const db = getDb();
+      return queryDoubleTexts(db, filters);
+    }
+  );
+
+  ipcMain.handle(
+    'query-left-on-read',
```

Comment:
oh this is is ACTUALLY being left on read. Ha. Took me a second to realize this was not a core query or something like that.